libcontainer/criu_linux.go

```diff
@@ -67,13 +69,13 @@ func (c *Container) checkCriuFeatures(criuOpts *CriuOpts, rpcOpts *criurpc.CriuO
 	if (criuFeat.LazyPages != nil) &&
 		(criuFeatures.LazyPages != nil) {
 		if *criuFeat.LazyPages && !*criuFeatures.LazyPages {
-			missingFeatures = true
+			missingFeatures = append(missingFeatures, "LazyPages")
 			logrus.Debugf("CRIU does not support LazyPages")
 		}
 	}
 
-	if missingFeatures {
-		return errors.New("CRIU is missing features")
+	if len(missingFeatures) != 0 {
+		return fmt.Errorf("%w: missing %v", ErrCriuMissingFeatures, missingFeatures)
```

Comment:
Nit: Would it be nicer to make this an error struct?

Reply:
Probably overengineering

Reply:
Yeah, fair enough.

Reply:
nit: the error message will look like this

```
criu is missing features: missing [MemTrack LazyPages]
```

I think the second "missing" is redundant.

IOW I suggest having

```go
return fmt.Errorf("%w: %v", ErrCriuMissingFeatures, missingFeatures)
```

Reply:
done